### ISSUE TITLE
Key Combination forwarding number to GS Click to Call

### ISSUE THREAD

**Issue by LennyM8472** [jabelone/grandstreamClickToCall#2]
Hi,

is it possible to select a phonenumber (any format) and by using a kay combination (like shift - c) the selected number will be forwarded to GS Click to Call?
Or as an alternative select a phone number and right click and there have the option to make a call?

Best regards
Lenny

**Comment by jabelone** [comment]
Hi, that was one of the original features I wanted to implement. However I've stopped working for the company and no longer have access to grandstream phones to test on. I'll leave the issue open in case anyone else wants to contribute.

**Comment by LennyM8472** [comment]
ah, very unfortunate

**Comment by jabelone** [comment]
Do you use it a lot? If it's super useful for you I may implement it when I get some free time. It was actually next on my list of things to improve this. :)

**Comment by LennyM8472** [comment]
Yes I do use it alot. The time saved not having to dial a number is so great. I will likely implement it on other Grandstream phones within my office as well.

Thanks very much appreciated.

Best regards
Lenny from Germany

**Comment by exelsmart** [comment]
dear jabelone.
Thank you very much for your addon to Chrome.
Please help add these functions to the program. You will help many people!
Also in the latest version of Chrome, the links "tel:" do not work
Please, help.
Maybe i can pay for your work? ...
I can give you access to the WEB UI remotely.
6733252@gmail.com
-----------------------------------------

Dmitry from Russia ;)
